pennylane/pauli/utils.py

```diff
@@ -66,8 +66,7 @@ def is_pauli_word(observable):
 
 
     Args:
-        observable (Observable): an observable, either a :class:`~.Tensor` instance or
-            single-qubit observable.
+        observable (Union[~.Observable]): the operator to be examined
```

Comment:
No need of `Union` if we only have one type

Reply:
Correct, I think this is a relic from another PR that I split to create this one 😄